diff_result.py

```diff
@@ -0,0 +1,30 @@
+from collections import defaultdict
+class DiffResult:
+    def __init__(self):
+        self.files = []
+        self.regions = 0
+        self.lineAdded = 0
+        self.lineDeleted = 0
+        self.functionCalls = defaultdict(int)
+
+    def toText(self):
+        with open('diffResult.txt', 'w') as output:
+            output.write("Files: \n")
```

Comment:
When testing, files are mislabeled as you left the `a/` and `b/` prefixes, most of them are doubled

Reply:
I noticed this but was not sure if this was the intended result or if the common sub-path was to be presented. This will be fixed.